### DISCUSSION TITLE
기본환경 셋팅법..

### DISCUSSION THREAD

**Discussion by disabled-variable** [storycraft/node-kakao#676] [General]
node-kakao를 이용해서 이것저것 개발도해보고 연습도해보려고합니다.
해당 실습을위해 어떻게 준비를해야할까요..
하나도 모르겠습니다.

현재 vscode 설치, nodejs 설치, 프로젝트 폴더 생성후 node-kakao 인스톨까지 완료했습니다.
기본적인 셋팅법 알려주실 천사개발자분 계신가요...ㅠㅠ

**Comment by benesiaxd** [comment]
소스코드에 모든게 다있습니다.
